src/components/power-sensor/client.test.ts

```diff
@@ -7,9 +7,9 @@ import { PowerSensorServiceClient } from '../../gen/component/powersensor/v1/pow
 import { PowerSensorClient } from './client';
 
 let sensor: PowerSensorClient;
-const testPower = { Watts: 0.5 };
-const testVoltage = [{ Volts: 1 }, { IsAc: true }];
-const testCurrent = [{ Amperes: 1 }, { IsAc: true }];
+const testPower = 0.5;
+const testVoltage = [1.5, true];
+const testCurrent = [1, true];
```

Comment:
[nit] might be a bit cleaner / DRYer if you make these test variables refer to the individual values and have your mock functions return them as well. i.e.

```suggestion
const testPower = 0.5;
const testVoltage = 1.5;
const testCurrent = 1;
const testIsAc = true
```